File: FirstBitSolutions/C Module/Assignments/Assignment 3/LoopsToNested/Ques7.c
//7 Find factorial of given number.
//Input: n = 5
//Output: 120
#include<stdio.h>
void main() {
	int n=5;
	int i,fact=1;
	for(i=n; i>1; i--) {
		fact = fact* i;
	}
	printf("%d is Factorial number %d",n,fact);
}

File: FirstBitSolutions/C Module/CtoJava Module/Assignment 3/Ques7.java
class Ques{ 
public static void main(String args[]){ 

	int n=5;
	int i =n,factorial=1;
	while(i>1){
		factorial = factorial* i;
		i--;
		
	}
	System.out.printf("%d is Factorial number %d",n,factorial);
 }
}